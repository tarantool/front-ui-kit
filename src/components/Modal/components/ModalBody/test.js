import React from 'react';
import renderer from 'react-test-renderer';

import { ModalBody } from '.';

describe('ModalBody', () => {
  it('renders without scrollbars', () => {
    const tree = renderer.create(
      <ModalBody className='external-class'>
        content
      </ModalBody>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it('renders with scrollbars', () => {
    const tree = renderer.create(
      <ModalBody className='external-class' scrollable>
        content
      </ModalBody>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
