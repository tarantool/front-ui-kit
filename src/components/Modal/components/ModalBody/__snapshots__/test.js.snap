// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ModalBody renders with scrollbars 1`] = `
.emotion-0 {
  position: relative;
  width: 100%;
  height: 100%;
}

.emotion-1 {
  padding-right: 30px;
  position: absolute;
  inset: 0px;
  overflow: auto;
}

.emotion-1 {
  scrollbar-width: thin;
  scrollbar-color: rgba(92, 96, 108, 0.5) transparent;
}

.emotion-1::-webkit-scrollbar {
  width: 6px;
  height: 6px;
  margin-right: 3px;
}

.emotion-1::-webkit-scrollbar-track {
  background-color: transparent;
}

.emotion-1::-webkit-scrollbar-thumb {
  background-color: rgba(92, 96, 108, 0.5);
  border-radius: 4px;
}

<div
  className="emotion-0 external-class"
>
  <div
    className="emotion-1"
  >
    content
  </div>
</div>
`;

exports[`ModalBody renders without scrollbars 1`] = `
.emotion-0 {
  width: 100%;
  height: 100%;
  padding-left: 30px;
  padding-right: 30px;
  overflow: hidden;
}

<div
  className="emotion-0 external-class"
>
  content
</div>
`;
